test/middleware/authentication.middleware.spec.ts

```diff
@@ -174,8 +175,10 @@ describe('Authentication middleware', () => {
   test("should redirect to signin page for update", async () => {
     const signinRedirectPath = `/signin?return_to=${SECURE_UPDATE_FILTER_URL}`;
 
-    mockIsActiveFeature.mockReturnValueOnce(false);
-    mockIsActiveFeature.mockReturnValueOnce(true);
+    mockIsActiveFeature.mockReturnValueOnce(false); // SHOW_SERVICE_OFFLINE_PAGE
+    mockIsActiveFeature.mockReturnValueOnce(false); // FEATURE_FLAG_ENABLE_REDIS_REMOVAL
+    mockIsActiveFeature.mockReturnValueOnce(false); // FEATURE_FLAG_ENABLE_REDIS_REMOVAL
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE_FLAG_ENABLE_REDIS_REMOVAL
```

Comment:
Like @dlloyd1-ch , also confused. Why would the value of the Redis feature flag ever change from being `false` to `true` within the scope of a single request-response cycle?

Reply:
it's the same for every test scope. the last one was unnecessary which is why the test was passing. fixed now.